roles/builder/handlers/main.yml

```diff
@@ -1,2 +1,16 @@
 ---
 # handlers file for builder
+
+- name: clear the package cache
+  ansible.builtin.file:
+    path: /var/cache/osbuild-composer/rpmmd
+    state: absent
+  listen:
+    - __clear_package_cache
+
+- name: restart services
+  ansible.builtin.systemd:
+    name: osbuild-composer.service
+    state: restarted
+  listen:
+    - __restart_services
```

Comment:
I'd like to suggestion renaming this `__restart_osbuild_composer_service` ... it's more verbose but it makes it more clear of what exactly it does in other task files within the role.

Reply:
Agreed, and actually I should have two things in the handlers file that listen on the same string, that'd be a cleaner way to do this I think

Reply:
@maxamillion have a look at the commits below, better names?